web/src/features/charts/elements/AreaGraph.tsx

```diff
@@ -23,13 +23,24 @@ const Y_AXIS_WIDTH = 40;
 const Y_AXIS_PADDING = 4;
 
 const getTotalValues = (layers: any) => {
```

Comment:
We should try and create a interface for layer (or layers) in a separate PR as we use it in several places.